### PR TITLE
simplify schema

### PR DESCRIPTION
**Pull request by zaqk**

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -179,9 +134,9 @@ enum VaultTransactionType {
 }
 
 type VaultTransaction @entity {
-  id: ID! #vault-uid
+  id: ID!  # os-vault-uniqueEventId
   os: DefaultOS!
-  epoch: Int!
+  epochNumber: Int!
```
Comment (tsully) [inline schema.graphql:139]: Does this need to be a derived field?

Also, curious if there's any additional value in linking the Epoch entity here. Just an idea - not sure if there is

**Review by tsully — APPROVED** [review]
LGTM - I'm happy we're simplifying the schema a bit. I think it will make it a lot easier to iterate on. 

Only comment is that I thought we were going to link the Epoch Entity on the other entities rather than just the epochNumber.

File: schema.graphql (hunk 1 above)
Comment (zaqk) [reply schema.graphql:139]: derived field wont help in this case.heres more info on that:
https://thegraph.com/docs/developer/create-subgraph-hosted#reverse-lookups

just left a message in the discord channel with some thoughts.

**Comment by zaqk** [comment]
merging as is based off this discussion:
https://discord.com/channels/872269102092263465/885034088380121120/890642226022072391

**Pull request merged**